tasks/Debian.yml

```diff
@@ -78,21 +78,6 @@
       state: latest
     become: yes
 
-  - name: Set up redis host name
-    shell: "{{ json }} -I -e 'this.services.CoAuthoring.redis.host  = \"{{ redis_server_host }}\"'"
-    when: redis_server_host != ""
-    notify: restart-ds
-
-  - name: Set up redis port number
-    shell: "{{ json }} -I -e 'this.services.CoAuthoring.redis.port  = \"{{ redis_server_port }}\"'"
-    when: redis_server_port != ""
-    notify: restart-ds
-
-  - name: Set up rabbitmq url
-    shell: "{{ json }} -I -e 'this.rabbitmq.url = \"{{ rabbitmq_server_url }}\"'"
-    when: rabbitmq_server_url != ""
-    notify: restart-ds
-
```

Comment:
I disagree with this change. It removes Redis and RabbitMQ configuration. So DocumentServer will use the default value `localhost` for those servers.
To escape error on this step use built-it `json` tool located at `/var/www/onlyoffice/documentserver/npm/json`

Reply:
okay, i misunderstood the code in the package